config/default/core.entity_view_display.node.localgov_directory.default.yml

```diff
@@ -25,6 +25,11 @@ content:
     third_party_settings: {  }
     weight: 0
     region: content
+  content_moderation_control:
```

Comment:
I believe they wish to hide the content moderation controls. I have noticed previously it adds itself in if not already hidden

Reply:
I've used the database from the Public Site channel 20230306_ecc_gov_prod.sql.gz
Is this the right one?